main.rb

```diff
@@ -51,13 +30,16 @@ def h(text)
 end
 
 post '/memos' do
-  memo = { 'id' => build_id, 'title' => params['title'], 'content' => params['content'], 'created_at' => Time.now }
-  save_memo(memo['id'], memo)
-  redirect "/memos"
+  id = memo.build_id
+  title = params['title']
+  content = params['content']
+  created_at = Time.now
+  memo.create(id, title, content, created_at)
+  redirect '/memos'
 end
 
 get '/memos/:id' do |id|
-  @memo = get_memo(id)
+  @memo = memo.show(id)
```

Comment:
```suggestion
  @memo = Memo.find(id)
```

この時点で何かが表示されるわけではないのでfindみたいな名前の方がいいかと思います